massa-models/src/mapping_grpc.rs

```diff
@@ -0,0 +1,201 @@
+use crate::block::{Block, FilledBlock, SecureShareBlock};
+use crate::block_header::{BlockHeader, SecuredHeader};
+use crate::endorsement::{Endorsement, SecureShareEndorsement};
+use crate::operation::{Operation, OperationType, SecureShareOperation};
+use crate::slot::{IndexedSlot, Slot};
+use massa_proto::massa::api::v1::{self as grpc, BytesMapFieldEntry, FilledOperationTuple};
+
+impl From<Block> for grpc::Block {
+    fn from(value: Block) -> Self {
+        grpc::Block {
+            header: Some(value.header.into()),
+            operations: value
+                .operations
+                .into_iter()
+                .map(|operation| operation.to_string())
+                .collect(),
+        }
+    }
+}
+
+impl From<BlockHeader> for grpc::BlockHeader {
+    fn from(value: BlockHeader) -> Self {
+        let mut res = vec![];
+        for endorsement in value.endorsements {
+            res.push(endorsement.into());
+        }
```

Comment:
It's probably prettier with iterators, same as parents? Something like
```rust
value.endorsements.into_iter().map(|e| e.into()).collect();
```

Reply:
done

Reply:
done